ogb_lsc/mag/split_and_save_indices.py

```diff
@@ -46,5 +46,5 @@ def main(argv) -> None:
 
 
 if __name__ == '__main__':
-  flags.mark_flag_as_required('root')
+  # flags.mark_flag_as_required('root')
```

Comment:
Could we keep this as `flags.mark_flag_as_required('data_root')`

Reply:
OK